packaging/resource_suite_s3_nocache.py

```diff
@@ -2356,6 +2357,42 @@ def test_upload_very_large_file_after_redirect_issues_2104_2114(self):
             s3plugin_lib.remove_if_exists(file1)
             s3plugin_lib.remove_if_exists(retrieved_file)
 
+    def test_iput_with_N_flag_issue_2198(self):
+
+        filename = 'test_iput_with_N_flag_issue_2198' 
+        get_filename = '%s.get' % filename
+        try:
+            file_sizes = [
+                    100,
+                    32 * 1024 * 1024 - 1,
+                    32 * 1024 * 1024,
+                    32 * 1024 * 1024 + 1,
+                    100 * 1024 * 1024
+                    ];
+
+            for file_size in file_sizes:
+                # create the file
+                lib.make_arbitrary_file(filename, file_size)
+
+                # put the file
+                self.user1.assert_icommand('iput %s' % filename)
+                self.user1.assert_icommand('iget -f %s %s' % (filename, get_filename))
+                assert_command(['diff', '-q', filename, get_filename])
+
+                for n_value in [2,3,4,8]:
```

Comment:
do we want to add a 0 and 1 to this array?

Reply:
done